<launch>
<arg name="robot_name" default="asl_gremlin1" />

<node   ns="$(arg robot_name)"
        name="trajectory_generator" 
        pkg="trajectory_generation" 
        type="trajectory_generator_node" 
        respawn="true" 
        output="screen"/>

</launch>
